1 Presentation layer/CandidateEvaluator.Client/Pages/Category/Category.cshtml

```diff
@@ -32,7 +32,7 @@ protected override async Task OnInitAsync()
 
 private async Task Save()
 {
-    await Http.AuthorizedPostJsonAsync<Contract.Models.Category>($"api/Category/{Id}", _categoryModel);
+    await Http.AuthorizedPostJsonAsync<Guid>($"api/Category/{Id}", _categoryModel);
```

Comment:
I believe that to be clear with `Get on Post` rule, you should get the id from post output and then redirect/navigate to the new path. Like in a CreateCategory page.